src/nxt_otel.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) F5, Inc.
+ */
+
+#ifndef _NXT_OTEL_H_INCLUDED_
+#define _NXT_OTEL_H_INCLUDED_
+
+#if (NXT_HAVE_OTEL)
+
+#include <nxt_router.h>
+
+// forward declared
+struct nxt_http_field_t;
+struct nxt_conf_validation_t;
+struct nxt_conf_value_t;
```

Comment:
Is this because including relevant header files causes issues?

Reply:
Admittedly, our header includes situation is a bit of a mess...

Reply:
yes.